packages/ui-components/src/components/Matrix/index.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+export { Matrix } from './Matrix';
+export * from './utils';
```

Comment:
Do the utils need to be exported? I can't see them used anywhere else.

Reply:
I mainly did this in case of using them in `tupaia-web` but I can remove until that comes up